themes/digital.gov/src/scss/new/_card-event.scss

```diff
@@ -232,6 +232,7 @@
 // Past events
 // ==========================================================================
 .card-event--past {
+  width: 100%;
```

Comment:
@clmedders part of why this is happening is the template.

If you look at `themes/digital.gov/layouts/topics/list.html` you'll see divs with the `grid-row` class **without** any grid children.

https://github.com/GSA/digitalgov.gov/blob/185b38fed524a6893d5575cb091f724c2a71cc13/themes/digital.gov/layouts/topics/list.html#L124-L129

Can we remove `grid-row` where there aren't grid children in this template?

Reply:
ahh that makes sense.. thank you for pointing that out.  I removed the width style and the grid-row for no grid children